docs/index.md

```diff
@@ -419,7 +419,7 @@ describe('Array', function() {
 });
 ```
 
-Pending tests will be reported as such.
+Such incomplete tests will still be included in the test results but as a pending test and not as a failure.
```

Comment:
Hmm, this is a bit of a run-on sentence.  Maybe:

"Pending tests will be included in the test results, and marked as pending.  A pending test is not considered a failed test."